openproblems/tasks/_batch_integration/batch_integration_graph/methods/seurat_full.py

```diff
@@ -0,0 +1,128 @@
+# from ....tools.normalize import log_cpm
```

Comment:
clean commented out code, and move to different PR. This file is not tested here